backend/src/api/model/series.rs

```diff
@@ -327,6 +349,193 @@ impl Series {
             out
         }).await
     }
+
+    pub(crate) async fn update_acl(id: Id, acl: Vec<AclInputEntry>, context: &Context) -> ApiResult<Series> {
+        if !context.config.general.allow_acl_edit {
+            return Err(err::not_authorized!("editing ACLs is not allowed"));
+        }
+
+        let series = Self::load_for_api(
+            id,
+            context,
+            err::invalid_input!(
+                key = "series.acl.not-found",
+                "series not found",
+            ),
+            err::not_authorized!(
+                key = "series.acl.not-allowed",
+                "acl update not allowed",
+            )
+        ).await?;
+
+        info!(series_id = %id, "Requesting ACL update of series");
+
+        let response = context
+            .oc_client
+            .update_acl(&series, &acl, context)
+            .await
+            .map_err(|e| {
+                error!("Failed to send acl update request: {}", e);
+                err::opencast_unavailable!("Failed to send acl update request")
```

Comment:
I guess ACL should be uppercase in both cases, and also change the same in `Event::update_acl`